dashboard/lti_new.py

```diff
@@ -16,19 +16,22 @@
 
 
 logger = logging.getLogger(__name__)
+INSTRUCTOR = 'Instructor'
+TA = 'TeachingAssistant'
+COURSE_MEMBERSHIP = 'membership'
```

Comment:
This is good.  I was thinking that these values should be in constants/variables.  They'd be helpful if the names of the roles ever change in the future.  I was hoping that maybe the "pylti1p3" module would have these defined already.

Ideally, the full role URIs would be used here.  It's better to be explicit.  We don't know whether Instructure will start sending some new role URI, like `http://instructure.com/role/membership#FUBAR-Instructor`, which we don't want, but would match the simple `Instructor` string this code is looking for.